core/src/main/scala/org/apache/spark/rpc/netty/NettyRpcEnv.scala

```diff
@@ -305,7 +305,7 @@ private[netty] class NettyRpcEnvFactory extends RpcEnvFactory with Logging {
   }
 }
 
-private[netty] class NettyRpcEndpointRef(@transient conf: SparkConf)
+private[netty] class NettyRpcEndpointRef(conf: SparkConf)
```

Comment:
`conf` is already defined as a transient field in the base class (`RpcEndpointRef`). Here it is only "forwarded" as a parameter, hence transient has no effect, hence we get a warning, hence the build fails